user-org-keycloak-util/src/main/java/org/sunbird/sso/service/ISSOService.java

```diff
@@ -0,0 +1,112 @@
+/** */
+package org.sunbird.sso.service;
+
+import org.sunbird.sso.model.User;
+
+/**
+ * This interface will have different method related to keycloak
+ *
+ * @author manzarul
+ */
+public interface ISSOService {
+
+  /**
+   * This method will verify user access token and provide userId if token is valid. in case of
+   * invalid access token it will throw ProjectCommon exception with 401.
+   *
+   * @param token String JWT access token
+   * @throws Exception
+   * @return String after successful verification it will return userId.
+   */
+  String verifyToken(String token) throws Exception;
+
+  /**
+   * This method will allow user to update password.
+   *
+   * @param userId String
+   * @param password String
+   * @return boolean
+   */
+  boolean updatePassword(String userId, String password);
+
+  /**
+   * Method to update user account in keycloak on basis of userId.
+   *
+   * @param user User
+   * @return boolean
+   */
+  boolean updateUser(User user) throws Exception;
```

Comment:
if we are using federated user database then this method is not required.

Reply:
you are correct we are using fedrated user , but these are the endpoint already used in sunbird, so i don't want to remove it.